quick-start/src/main/java/com/marklogic/hub/factory/EntityModelFactory.java

```diff
@@ -95,6 +95,9 @@ public EntityModel createEntity(String entityName, String entityFilePath) {
                 + flowType.toString();
         List<String> flowNames = FileUtil.listDirectFolders(flowsFilePath);
         for (String flowName : flowNames) {
+            if (flowName.equals("REST")) {
```

Comment:
Not clear from this snippet why you'd skip a REST flow. A comment would help, unless it's clear from broader context.